Makefile

```diff
@@ -7,7 +7,15 @@ npm-deps:
 	cd client && npm install
 
 grunt: npm-deps
-	cd client && grunt
+	cd client && node_modules/grunt-cli/bin/grunt
+
+style: npm-deps
+	cd client && node_modules/grunt-cli/bin/grunt style
+
+webpack: npm-deps
+	cd client && node_modules/webpack/bin/webpack.js -p
```

Comment:
awesome, thanks for documenting essentially how to do this. This is excellent! :)